pkg/controllers/disruption/metrics.go

```diff
@@ -29,6 +29,17 @@ const (
 	decisionLabel                = "decision"
 	ConsolidationTypeLabel       = "consolidation_type"
 	CandidatesIneligible         = "candidates_ineligible"
+
+	// Label constants
+	ClusterCostLabel              = "cluster_cost"
+	TotalCPURequestsLabel         = "total_cpu_requests"
+	TotalMemoryRequestsLabel      = "total_memory_requests"
+	TotalNodeCountLabel           = "total_node_count"
+	TotalDesiredPodCountLabel     = "total_desired_pod_count"
+	InvolvesPodAntiAffinityLabel  = "involves_pod_anti_affinity"
+	NodePoolNameLabel             = "nodepool_name"
+	PodCPURequestChangeRatioLabel = "pod_cpu_request_change_ratio"
+	PodMemRequestChangeRatioLabel = "pod_mem_request_change_ratio"
```

Comment:
Do we want to consider abstracting this to support any resource type supported by a node? i.e. gpu reservation is very interesting.